operator/watchers/cilium_endpoint.go

```diff
@@ -163,3 +211,40 @@ func HasCE(ns, name string) (*cilium_api_v2.CiliumEndpoint, bool, error) {
 	cep := item.(*cilium_api_v2.CiliumEndpoint)
 	return cep, exists, nil
 }
+
+func endpointUpdated(cep *cilium_api_v2.CiliumEndpoint) {
```

Comment:
I think this function should have the "ceb" or "ciliumendpointbatch" in its name somewhere...

Maybe notifyCEBOnEndpointUpdated or something?

Reply:
This function called for every `CEP` updates in a cluster.  This function name should be generic, there is a check inside this function  to check if `cebController` is enabled.